_pages/docs/matching-options.md

```diff
@@ -124,6 +128,31 @@ The `reduce` element gives a way to back off the scores in such cases. The
 element under `algorithms`. The `weight` attribute will be subtracted from the
 score if the algorithm matches.
 
+### Actions
+
+You can provide your own custom action handlers. These get called when a match is made. Smart-mastering-core provides two default action handlers: notify and merge.
```

Comment:
"Custom actions are called after merge and notify have run. These actions will be called in a separate transaction, so the results of each call will be visible to the actions called after it." (Feel free to reword)